avaframe/runScripts/runStandardTestsCom1DFAOrig.py

```diff
@@ -87,8 +85,8 @@
 
     # Load input parameters from configuration file for standard tests
     # write config to log file
-    avaName = os.path.basename(avaDir)
-    standardCfg = os.path.join('..', 'benchmarks', test['NAME'], '%s_com1DFAOrigCfg.ini' % test['AVANAME'])
+    avaName = pathlib.Path(avaDir).name
```

Comment:
Is this actually needed, since the AvaName also lives in test['AVANAME']?

Reply:
removed